extension/war/src/main/webapp/WEB-INF/conf/social-extension/social/core-configuration.xml

```diff
@@ -676,6 +676,23 @@
           <value>org.exoplatform.social</value>
         </value-param>
       </init-params>
+    </component-plugin>  
+  </external-component-plugins>
+  
+  <external-component-plugins>
```

Comment:
Is it intended to not remove the other UpgradePlugin (ModelOverriderUpdaterPlugin) ? Where is specified the list of upgrade plugins to enable ?